pkg/generate/git/git.go

```diff
@@ -21,13 +21,14 @@ import (
 // - ssh
 func ParseRepository(s string) (*url.URL, error) {
 	uri, err := url.Parse(s)
-	if err != nil {
-		return nil, err
+	if err == nil && uri.Scheme != "" {
```

Comment:
I would expect some breakage here, but maybe the test isn't covering what I'm expecting.  It looks like now the error from `url.Parse` is being completely ignored which doesn't seem right?  I would've expected that to break the test somehow.